content/guides/use-case/jupyter/customize.md

```diff
@@ -0,0 +1,261 @@
+environment with others, or to use it as a base for further development.
+
+## Prerequisites
+
+- You have installed the latest version of [Docker Desktop](../../../get-docker.md).
+- Optionally, you have completed [Run JupyterLab](run-jupyter.md) for more
+  context.
+
+## Identify your environment
+
+For this example, you'll use the Iris Dataset example from
+[Run JupyterLab](run-jupyter.md). When you run a new container and then run that
+notebook, the dependencies, `matplotlib` and `scikit-learn`, need to be
+installed every time. While the dependencies in that small example download and
+install quickly, it may become a problem as your list of dependencies grow.
+There may also be other tools, packages, or files that you always want in your
+environment.
+
+In this case, you can install the dependencies as part of the environment in the
+image. Then, every time you run your container, the dependencies will always be
+installed.
+
+Once you have identified your environment, you can start defining it in a
+Dockerfile. A Dockerfile is a text file that instructs Docker how to create an
+image of your JupyterLab environment. An image contains everything you want and
+need when running JupyterLab, such as files, packages, and tools.
+
+## Define your environment in a Dockerfile
+
+In a directory of your choice, create a new text file named `Dockerfile`. Open the `Dockerfile` in an IDE or text editor and then add the following contents.
+
+```dockerfile
+# syntax=docker/dockerfile:1
+
+FROM jupyter/base-notebook
+RUN pip install --no-cache-dir matplotlib scikit-learn
+```
+
+Before you proceed, save your changes to the `Dockerfile`.
+
+The following is a line-by-line breakdown of the `Dockerfile`.
+
+
+```dockerfile
+# syntax=docker/dockerfile:1
+```
+
+This optional instruction is a parser directive that tells Docker to use a
+specific Dockerfile syntax for building the image. The syntax is defined by an
+external image that contains the instructions and features for the Dockerfile.
+Different versions of the Dockerfile syntax may have different features and
+instructions that you can use to customize your image. By using a parser
+directive, you can ensure that your Dockerfile is interpreted correctly by the
+Docker Engine, and avoid potential errors or inconsistencies.
+
+```dockerfile
+FROM jupyter/base-notebook
+```
+
+This `FROM` instruction specifies the base image to use for building your own image. The base image is an existing image that has some of the files, packages, and tools you need for your JupyterLab environment. By using a base image, you can save time and avoid repeating common steps in your Dockerfile.
+
+```dockerfile
+RUN pip install --no-cache-dir matplotlib scikit-learn
+```
+
+The `RUN` instruction executes commands on top of the current image. In this
+case, it's running `pip install --no-cache-dir matplotlib scikit-learn` to install dependencies. You can also use the `RUN` instruction to install JupyterLab extensions.
```

Comment:
suggestion: similar to my comment about the `docker run` command, could we make this shorter and more digestable? For example, maybe we don't need to cover parser directives here.